Mathlib/Combinatorics/SimpleGraph/Copy.lean

```diff
@@ -356,6 +360,205 @@ alias ⟨IsIndContained.exists_iso_subgraph, IsIndContained.of_exists_iso_subgra
 @[simp] lemma compl_isIndContained_compl : Gᶜ ⊴ Hᶜ ↔ G ⊴ H :=
   Embedding.complEquiv.symm.nonempty_congr
 
-protected alias ⟨IsIndContained.of_compl, IsIndContained.compl⟩ := compl_isIndContained_compl
+/-!
+### Counting the copies
+
+If `G` and `H` are finite graphs, we can count the number of unlabelled and labelled copies of `G`
+in `H`.
+
+#### Not necessarily induced copies
+-/
+
+section CopyCount
+variable [Fintype V]
+
+/-- `G.copyCount H` is the number of unlabelled copies of `H` in `G`.
+See `SimpleGraph.labelledCopyCount` for the number of labelled copies. -/
+noncomputable def copyCount (G : SimpleGraph V) (H : SimpleGraph W) : ℕ := by
+  classical exact #{G' : G.Subgraph | Nonempty (H ≃g G'.coe)}
```

Comment:
How bad is it to make this computable?

Reply:
Computable, probably not so bad, but seems out of scope writing the necessary decidability instances. Efficiently computable, I would say quite hard, because you're basically asking me to implement a graph isomorphism algorithm.

Reply:
Efficiently computable is definitely not reasonable here. But let's have a TODO to make it computable - probably the decidability instances that come out will be independently useful